solutions/usaco-531.mdx

```diff
@@ -122,3 +122,85 @@ int main()
 </CPPSection>
 
 </LanguageSection>
+
+
+## Prim's Implementation
+
+	
+### Explanation
```

Comment:
should be `## Explanation`, move the header and explanation above the Implementation header